### PR TITLE
i.e. changed to e.g.

### LINKED ISSUE #677: Replace "i.e." with "e.g." where needed
In some places (docs, UI and other text files) we use the abbreviation "i.e." where "e.g." actually makes more sense.
"e.g." means "For example", while "i.e." means "That is".

Find all places where we use "i.e." and replace it (only if applicable)

Clarification edit: the issue here is to find the places where "i.e." appears *by mistake*, not all of its appearances.

### PR DESCRIPTION
**Pull request by Yashs911**
Fixes #677

### CODE REVIEW

**Comment by Yashs911** [comment]
@arielshaqed I am new to open source and I think I made some mistake here. If you guide me on how I can pass all the checks.
If by any chance this PR is not up to the standard I am ready to close this.
Sorry if I caused any inconvenience.

**Comment by arielshaqed** [comment]
> @arielshaqed I am new to open source and I think I made some mistake here. If you guide me on how I can pass all the checks.
> If by any chance this PR is not up to the standard I am ready to close this.
> Sorry if I caused any inconvenience.

Don't worry about the failing test, its an integration test that cannot work for external contributors.  I've asked our @johnnyaug to review this code.  It's a holiday weekend over here, our response time is a bit slower.

File: docs/branching/recommendations.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ For our commit, let's also add the Git commit hash for the job's source code and
 
 This is similar to the previous case, except we now have several jobs that produce related data. We want to be able to release all related data as one atomic unit, or revert them all together (or partially) if something goes wrong.
 
-A common use case for this are "materialized views" - i.e. the same data but partitioned by different columns or sorted differently to optimize for different readers.
+A common use case for this are "materialized views" - e.g. the same data but partitioned by different columns or sorted differently to optimize for different readers.
```
Comment (johnnyaug) [inline docs/branching/recommendations.md:85]:
```suggestion
A common use case for this are "materialized views" - i.e. the same data but partitioned by different columns or sorted differently to optimize for different readers.
```

This one was actually correct

**Review by johnnyaug — COMMENTED** [review]
Hi @Yashs911 , thank you for your effort!
See my comment

**Comment by Yashs911** [comment]
@johnnyaug I have added a new commit with the required changes.

**Review by johnnyaug — APPROVED** [review]

**Comment by johnnyaug** [comment]
Thank you @Yashs911, merged!

**Pull request merged**